src/content/getting-started/architecture/networkplugin-syncer/_index.en.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Network Plugin Syncer"
+---
+
+The Network Plugin Syncer provides a framework for components to  interface
+with the configured Kubernetes CNI Network plugin to perform any high level
+API tasks necessary to facilitate routing cross-cluster traffic.
```

Comment:
maybe a nit but I'm not clear on what "_high level API_" means. Perhaps we can just omit this phrase and just say "_perform any tasks_" w/o and qualification?

Reply:
Hmm, yeah, but the route-agent will also perform "tasks", it's too generic. I'll try to explain a little bit more.

Reply:
Let me know if it looks more OK.

I want to express the difference with the route-agent which configures ip tables, ip routes/rules, etc...